### ISSUE TITLE
Production build fails: "You must return an object from `Filter.prototype.postProcess`"

### ISSUE THREAD

**Issue by sebastianhelbig** [ember-cli/ember-cli-clean-css#305]
ember-cli 5.1.0
ember-cli-clean-css 2.0.0

```
  …
  fast-sourcemap-concat: [SourceMapConcat#6198] encoder cache prime: @mvb/tix-ui/styles/publishers/index.css +0ms
  fast-sourcemap-concat: [SourceMapConcat#6198] encoder cache prime: @mvb/tix-ui/styles/reset-password.css +0ms
⠋ building... [Package /assets/vendor.js]  broccoli-persistent-filter:CleanCSSFilter [CleanCSSFilter#6199 -> derivePatches#6200] found 0 files invalidated due to dependency changes. +7ms
  broccoli-persistent-filter:CleanCSSFilter [CleanCSSFilter#6199 -> derivePatches#6200] derivePatches duration: 0.88 ms {"patches":4,"entries":4,"walk":{"entries":4,"duration":"0.61 ms"},"invalidations":{"dependencies":0,"count":0,"duration":"0.12 ms"}} +0ms
cleaning up...
  ember-cli:command An error occurred running `Build` from the `build` command. Error: You must return an object from `Filter.prototype.postProcess`.
    at Object.assertNever [as default] (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/util/assertNever.js:4:11)
    at DefaultStrategy.processString (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/strategies/default.js:17:34)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async invoke (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:84:12)
    at async CleanCSSFilter.processFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:398:22)
    at async CleanCSSFilter.processAndCacheFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:374:20)
    at async CleanCSSFilter._handleFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:277:26) +2s
Build Error (CleanCSSFilter) in assets/@mvb/tix-ui.css

You must return an object from `Filter.prototype.postProcess`.


Stack Trace and Error Report: /var/folders/hh/vn1vt7tn5b17n7hrn22zwwdr0000gn/T/error.dump.6a14d527ed7842c9b7dffa9256b37599.log
```

```
ERROR Summary:

  - broccoliBuilderErrorStack: Error: You must return an object from `Filter.prototype.postProcess`.
    at Object.assertNever [as default] (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/util/assertNever.js:4:11)
    at DefaultStrategy.processString (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/strategies/default.js:17:34)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async invoke (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:84:12)
    at async CleanCSSFilter.processFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:398:22)
    at async CleanCSSFilter.processAndCacheFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:374:20)
    at async CleanCSSFilter._handleFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:277:26)
  - code: [undefined]
  - codeFrame: You must return an object from `Filter.prototype.postProcess`.
  - errorMessage: assets/@mvb/tix-ui.css: You must return an object from `Filter.prototype.postProcess`.
        in /var/folders/hh/vn1vt7tn5b17n7hrn22zwwdr0000gn/T/broccoli-42176Lwb49mHlj8jr/out-607-source_map_concat
        at CleanCSSFilter
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: assets/@mvb/tix-ui.css
    - line: [undefined]
    - treeDir: /var/folders/hh/vn1vt7tn5b17n7hrn22zwwdr0000gn/T/broccoli-42176Lwb49mHlj8jr/out-607-source_map_concat
  - message: assets/@mvb/tix-ui.css: You must return an object from `Filter.prototype.postProcess`.
        in /var/folders/hh/vn1vt7tn5b17n7hrn22zwwdr0000gn/T/broccoli-42176Lwb49mHlj8jr/out-607-source_map_concat
        at CleanCSSFilter
  - name: Error
  - nodeAnnotation: [undefined]
  - nodeName: CleanCSSFilter
  - originalErrorMessage: You must return an object from `Filter.prototype.postProcess`.
  - stack: Error: You must return an object from `Filter.prototype.postProcess`.
    at Object.assertNever [as default] (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/util/assertNever.js:4:11)
    at DefaultStrategy.processString (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/strategies/default.js:17:34)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async invoke (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:84:12)
    at async CleanCSSFilter.processFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:398:22)
    at async CleanCSSFilter.processAndCacheFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:374:20)
    at async CleanCSSFilter._handleFile (/Users/sh/Developer/tix/node_modules/.pnpm/broccoli-persistent-filter@3.1.3/node_modules/broccoli-persistent-filter/lib/index.js:277:26)
```

**Comment by jkburges** [comment]
I was getting the same error, but release 3.0.0 seems to have fixed it for me.

**Comment by sebastianhelbig** [comment]
Also fixed for me in 3.0.0.
